catroid/src/org/catrobat/catroid/physics/PhysicsLook.java

```diff
@@ -59,6 +63,7 @@ public void setLookData(LookData lookData) {
 		super.setLookData(lookData);
 		PhysicsWorld physicsWorld = ProjectManager.getInstance().getCurrentProject().getPhysicsWorld();
 		physicsWorld.changeLook(physicsObject, this);
+		updatePhysicsObjectState(true);
```

Comment:
PhysicsObjectStateHandler.update(boolean) method doesn't need a parameter as well.
@Gorphi did some refactoring related to these methods. Maybe he can help.